roentgen/continuum/emission.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np
+
+import astropy.units as u
+from astropy.constants.codata2018 import h, c, e, k_B
+
+__all__ = [
+    "thermal_bremsstrahlung",
+    "mewe_gaunt",
+    "xray_tube_spectrum",
+    "cutoff_wavelength"
+]
+
+
+def thermal_bremsstrahlung(wavelength, z, density, temperature):
+    """
+    This equation assumes that the density of ions is equal to the density of
+    electrons.
+    """
+    frequency = wavelength.to('Hz', equivalencies=u.spectral())
```

Comment:
I think this should be `to_value()` as `np.exp` doesn't take dimensional quantities.

Reply:
actually as long as the units cancel out in `np.exp` then it works fine!

Reply:
I think there is a 's' left over from the frequency?